addons/mail/static/src/components/notification_list/notification_list.xml

```diff
@@ -30,7 +30,13 @@
                             notificationGroupLocalId="notification.notificationGroup.localId"
                         />
                     </t>
-                    <t name="separatorCondition" t-if="!notification_last">
+                    <t t-if="notification.type === 'odoobotRequest'">
+                        <NotificationRequest
+                            class="o_NotificationList_notificationRequest"
+                            t-att-class="{ 'o-mobile': env.messaging.device.isMobile }"
+                        />
+                    </t>
+                    <t t-if="!notification_last">
```

Comment:
don't like relying on magic variables of `t-foreach`, but we know we have to re-work notifications so could be done later.

Reply:
I just removed the `name` from that tag btw, this is exactly like before.

This is not related to notifications though, it's just a separator here.

The only "way" to get rid of this foreach variable is to have the separator modeled as part of the list.